database/connection.go

```diff
@@ -29,13 +32,11 @@ func (db *Database) Close() error {
 	return nil
 }
 
-func (db *Database) ConnectionStatus() (err error, ok bool) {
-	// Returns error if the file is not a valid database
-	
+func (db *Database) ConnectionStatus() (ok bool, err error) {
```

Comment:
Ok we can just make it return the error then. So if connection was fail, err will be set.. else err will be nil

Reply:
I believe a ok and error together is unnecessary.

Reply:
ok